### PR TITLE
Update advanced-hunting-query-results.md

### LINKED ISSUE #13161: page accessibility issue
It feels like it's an accessibility issue when the kusto syntax relevant to the page only appears in screenshots/pictures. There are examples of kusto queries in codeblocks, however the subsequent example for how to render it in a chart format only appears in images. Fix should be that the "| render " syntax should appear in text.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 5b6c8b2d-1fa1-0f6e-a287-bd5d534ea2a3
* Version Independent ID: 1eea0bf3-2432-d56d-016a-9f57f09e84ff
* Content: [Work with advanced hunting query results in Microsoft 365 Defender](https://learn.microsoft.com/en-us/microsoft-365/security/defender/advanced-hunting-query-results?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender/advanced-hunting-query-results.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender/advanced-hunting-query-results.md)
* Service: **microsoft-365-security**
* Sub-service: **m365d**
* GitHub Login: @schmurky
* Microsoft Alias: **maccruz**

### PR DESCRIPTION
**Pull request by msbemba**
Updated |render syntax as it was in screenshot

fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/13161

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@schmurky Please merge this PR. Thanks

File: microsoft-365/security/defender/advanced-hunting-query-results.md [hunk 1]
```diff
@@ -69,6 +69,13 @@ AlertInfo
 
 When rendering the results, a column chart displays each severity value as a separate column:
 
+
+```kusto
+AlertInfo
+| summarize Total = count() by Severity
+| render coloumnchart
```
Comment (schmurky) [inline microsoft-365/security/defender/advanced-hunting-query-results.md]: col**o**umnchart > columnchart

Once this is changed, it should be good to go.

**Review by schmurky — CHANGES_REQUESTED** [review]

**Comment by yogkumgit** [comment]
@msbemba

**Comment by msbemba** [comment]
Thank you @schmurky . I have made the changes as per the request.

**Review by schmurky — APPROVED** [review]

**Comment by schmurky** [comment]
[like]  Macky Cruz reacted to your message:
________________________________

**Pull request merged**